dascore/core/summary.py

```
@@ -0,0 +1,327 @@
+
+    def __getattr__(self, item):
+        try:
+            return getattr(self.attrs, item)
+        except AttributeError:
+            split = item.rsplit("_", 1)
+            if len(split) == 2:
+                coord_name, field = split
+                if coord_name in self.coords:
+                    return getattr(self.coords[coord_name], field, None)
+            msg = f"{self.__class__.__name__} has no attribute '{item}'"
+            raise AttributeError(msg)
```

Ruff (0.15.5)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)